#It's highly likely that all these calls have moved to ServerPlayer.
#We should re-apply this there. 
--- a/net/minecraft/world/entity/player/Player.java
+++ b/net/minecraft/world/entity/player/Player.java
@@ -1417,19 +_,19 @@
             int i = Math.round((float)Math.sqrt(p_36379_ * p_36379_ + p_36380_ * p_36380_ + p_36381_ * p_36381_) * 100.0F);
             if (i > 0) {
                this.awardStat(Stats.SWIM_ONE_CM, i);
-               this.causeFoodExhaustion(0.01F * (float)i * 0.01F);
+               this.causeFoodExhaustion(this.level().spigotConfig.swimMultiplier * (float)i * 0.01F, EntityExhaustionEvent.ExhaustionReason.SWIM); // CraftBukkit - EntityExhaustionEvent // Spigot
             }
          } else if (this.isEyeInFluid(FluidTags.WATER)) {
             int j = Math.round((float)Math.sqrt(p_36379_ * p_36379_ + p_36380_ * p_36380_ + p_36381_ * p_36381_) * 100.0F);
             if (j > 0) {
                this.awardStat(Stats.WALK_UNDER_WATER_ONE_CM, j);
-               this.causeFoodExhaustion(0.01F * (float)j * 0.01F);
+               this.causeFoodExhaustion(this.level().spigotConfig.swimMultiplier * (float)j * 0.01F, EntityExhaustionEvent.ExhaustionReason.WALK_UNDERWATER); // CraftBukkit - EntityExhaustionEvent // Spigot
             }
          } else if (this.isInWater()) {
             int k = Math.round((float)Math.sqrt(p_36379_ * p_36379_ + p_36381_ * p_36381_) * 100.0F);
             if (k > 0) {
                this.awardStat(Stats.WALK_ON_WATER_ONE_CM, k);
-               this.causeFoodExhaustion(0.01F * (float)k * 0.01F);
+               this.causeFoodExhaustion(this.level().spigotConfig.swimMultiplier * (float)k * 0.01F, EntityExhaustionEvent.ExhaustionReason.WALK_ON_WATER); // CraftBukkit - EntityExhaustionEvent // Spigot
             }
          } else if (this.onClimbable()) {
             if (p_36380_ > 0.0D) {
@@ -1440,13 +_,13 @@
             if (l > 0) {
                if (this.isSprinting()) {
                   this.awardStat(Stats.SPRINT_ONE_CM, l);
-                  this.causeFoodExhaustion(0.1F * (float)l * 0.01F);
+                  this.causeFoodExhaustion(this.level().spigotConfig.sprintMultiplier * (float)l * 0.01F, EntityExhaustionEvent.ExhaustionReason.SPRINT); // CraftBukkit - EntityExhaustionEvent // Spigot
                } else if (this.isCrouching()) {
                   this.awardStat(Stats.CROUCH_ONE_CM, l);
-                  this.causeFoodExhaustion(0.0F * (float)l * 0.01F);
+                  this.causeFoodExhaustion(this.level().spigotConfig.otherMultiplier * (float)l * 0.01F, EntityExhaustionEvent.ExhaustionReason.CROUCH); // CraftBukkit - EntityExhaustionEvent // Spigot
                } else {
                   this.awardStat(Stats.WALK_ONE_CM, l);
-                  this.causeFoodExhaustion(0.0F * (float)l * 0.01F);
+                  this.causeFoodExhaustion(this.level().spigotConfig.otherMultiplier * (float)l * 0.01F, EntityExhaustionEvent.ExhaustionReason.WALK); // CraftBukkit - EntityExhaustionEvent // Spigot
                }
             }
          } else if (this.isFallFlying()) {
